### PR TITLE
promoting error bullets to sections to HTML anchors work

### PR DESCRIPTION
**Pull request by miyamotoh**
Changing doc style from bullets for each error to headings/sections so pointing to a specific known error works with URL with HTML anchors (`#foo`).

Signed-off-by: Hiro Miyamoto <miyamotoh@us.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/103#" title="Author self-approved">miyamotoh</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpradipt**
You can assign the PR to them by writing `/assign @bpradipt` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/known_issues.md
Comment (bpradipt) [inline docs/known_issues.md:73]: Please delete this issue. This is currently taken care in deployment code itself

**Comment by bpradipt** [comment]
Thanks @miyamotoh .. While you are modifying this. Can you please add these additional issues.
For all these cases, the `apply` command need to be re-run.

```
module.install.null_resource.install (remote-exec): information.
module.install.null_resource.install (remote-exec): changed: [192.168.25.12] => {"ansible_facts": {"discovered_interpreter_python": "/usr/libexec/platform-python"}, "changed": true, "cmd": "if lsmod|grep -q 'ibmveth'; then\n  sudo sysctl -w net.ipv4.route.min_pmtu=1450;\n  sudo sysctl -w net.ipv4.ip_no_pmtu_disc=1;\n  echo 'net.ipv4.route.min_pmtu = 1450' | sudo tee --append /etc/sysctl.d/88-sysctl.conf > /dev/null;\n  echo 'net.ipv4.ip_no_pmtu_disc = 1' | sudo tee --append /etc/sysctl.d/88-sysctl.conf > /dev/null;\nfi\n", "delta": "0:00:00.078644", "end": "2020-09-18 16:25:46.414601", "rc": 0, "start": "2020-09-18 16:25:46.335957", "stderr": "", "stderr_lines": [], "stdout": "net.ipv4.route.min_pmtu = 1450\nnet.ipv4.ip_no_pmtu_disc = 1", "stdout_lines": ["net.ipv4.route.min_pmtu = 1450", "net.ipv4.ip_no_pmtu_disc = 1"]}

module.install.null_resource.install (remote-exec): NO MORE HOSTS LEFT *************************************************************

module.install.null_resource.install (remote-exec): PLAY RECAP *********************************************************************
module.install.null_resource.install (remote-exec): 192.168.25.105             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.12              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.121             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.15              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.235             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.39              : ok=1    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.5               : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.82              : ok=28   changed=22   unreachable=0    failed=0    skipped=11   rescued=0    ignored=0

```

```
module.install.null_resource.config (remote-exec): TASK [Downloading OCP4 Installer] **********************************************
module.install.null_resource.config: Still creating... [1m40s elapsed]
module.install.null_resource.config (remote-exec): fatal: [localhost]: FAILED! => {"changed": false, "dest": "/usr/local/src/openshift-install-linux.tar.gz", "elapsed": 10, "msg": "Request failed: <urlopen error _ssl.c:880: The handshake operation timed out>", "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/4.5.4/openshift-install-linux.tar.gz"}

```

```
module.install.null_resource.install (remote-exec): fatal: [192.168.25.167]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.25.167 port 22: Connection timed out", "unreachable": true}

module.install.null_resource.install (remote-exec): NO MORE HOSTS LEFT *************************************************************

module.install.null_resource.install (remote-exec): PLAY RECAP *********************************************************************
module.install.null_resource.install (remote-exec): 192.168.25.107             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.165             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.167             : ok=1    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.212             : ok=28   changed=22   unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.64              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.76              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.78              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
module.install.null_resource.install (remote-exec): 192.168.25.81              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```

File: docs/known_issues.md
Comment (bpradipt) [reply docs/known_issues.md:59]: @miyamotoh this is still there ..

File: docs/known_issues.md
Comment (bpradipt) [inline docs/known_issues.md:85]: Can we put some additional empty lines to make the distinction between different types of error?
Currently it looks like the entire message is part of same error..

File: docs/known_issues.md
Comment (miyamotoh) [reply docs/known_issues.md:59]: Oh, that whole MTU=1450 thing is becoming non-issue? Misunderstood. Will be in my next push...

File: docs/known_issues.md
Comment (miyamotoh) [reply docs/known_issues.md:85]: Adding bullet points...

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @miyamotoh

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/103#pullrequestreview-497019837" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/103#" title="Author self-approved">miyamotoh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bpradipt** [comment]
Can you also raise a PR against the release-4.5 branch ?

**Pull request merged**
